action.yml

```diff
@@ -9,11 +9,13 @@ runs:
 inputs:
   apiKey:
     description: "Your Cloudflare API Key"
-    required: true
+  apiToken:
+    description: "(Legacy) Your Cloudflare API Token"
```

Comment:
API Key is legacy, not token, correct?

Reply:
same. guess the api key was meant to be legacy?

Reply:
oh oops! thank you, good catch